Services/ProjectService/Controllers/ProjectController.cs

```diff
@@ -6,6 +7,10 @@
 using LT.DigitalOffice.Kernel.Broker;
 using LT.DigitalOffice.ProjectService.Models;
 using LT.DigitalOffice.ProjectService.Commands.Interfaces;
+using LT.DigitalOffice.ProjectService.Models;
```

Comment:
Chek warnings
"The using directive for 'LT.DigitalOffice.ProjectService.Models' appeared previously in this namespace"

Reply:
Fixed